packages/components/src/components/FileUploader/FileUploader.test.tsx

```diff
@@ -100,7 +100,13 @@ describe("<FileUploader />", () => {
 
       expect(screen.getByRole("link", { name: "Visa" })).toBeInTheDocument();
       expect(
-        screen.queryByText("No file uploaded • 1MB")
+        screen.queryByText("No file uploaded", { exact: false })
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByText("1MB", { exact: false })
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByText("2MB", { exact: false })
```

Comment:
Why add these checks?

Reply:
To make sure we don't show _any_ kind of file information. Before we only had the "1MB" which is the `fileSize`, but if you see the screenshot in the description it's also possible that we incorrectly show the `maxFileSize`. If we had had the "2MB" check then we might have caught this. I figured I'd do different expectations (not exact) to avoid having all possible combinations

Reply:
I don't think checking all of these texts is a good practice, but I understand the issue. We can't use a data-testid here, so, that's fine for me. But let's try to remove the `{ exact: false }` because we don't need them.